tools/slang-test/slang-test-main.cpp

```diff
@@ -1106,6 +1106,18 @@ TestResult runCompile(TestContext* context, TestInput& input)
         return TestResult::Pass;
     }
 
+    if (exeRes.resultCode != 0)
+    {
+        auto reporter = context->reporter;
+        if (reporter)
+        {
+            auto output = getOutput(exeRes);
+            reporter->message(TestMessageType::TestFailure, output);
```

Comment:
Were error messages from `COMPILE` tests really not getting reported somehow? I feel like I would have noticed...